server/routes/climb.router.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const {
+  rejectUnauthenticated,
+} = require('../modules/authentication-middleware');
+const pool = require('../modules/pool');
+const router = express.Router();
+
+/**
+ * GET route template
+ */
+router.get('/', (req, res) => {
+  queryText = `
+    SELECT "climbs".id as "climbId", "climbs".attempts, "climbs"."climbDate", "climbs".color, "climbs"."isSubmitted", "climbs".level,
+    "climbs"."userId", "locations".name AS "locationName"
+    FROM "climbs"
+    JOIN "locations" ON "climbs"."locationId" = "locations".id;
+  `
+  pool.query(queryText)
+  .then((dbRes) => {
+    res.send(dbRes.rows)
+  })
+  .catch((err) => {
+    console.log('Error in GET all climbs');
+    res.sendStatus(500);
+  })
+});
+
+/**
+ * POST route template
+ */
+router.post('/add', rejectUnauthenticated, (req, res) => {
```

Comment:
This can just be `.post('/')`. The fact that we're posting already tells us that we're "adding" something.